paddle/string/to_string_test.cc

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+#include "paddle/string/to_string.h"
+#include <gtest/gtest.h>
+
+constexpr char OUT_STR[] = "User Defined Output";
```

Comment:
https://google.github.io/styleguide/cppguide.html#Constant_Names

Variables declared **constexpr** or const, and whose value is fixed for the duration of the program, are named with a leading "k" followed by mixed case. For example:

```
const int kDaysInAWeek = 7;
```